catalog/networking/network/subnet/setters.yaml

```diff
@@ -3,10 +3,5 @@ kind: ConfigMap
 metadata:
   name: setters
 data:
-  namespace: networking
   ip-cidr-range: 10.2.0.0/16
-  network-name: network-name
-  project-id: project-id
-  region: us-central1
   source-subnetwork-ip-ranges-to-nat: ALL_SUBNETWORKS_ALL_IP_RANGES
-  prefix: ""
```

Comment:
Not sure I fully understand the UX. I would have expected to have N subnets with different prefixes and have no prefix in the [parent setter](https://github.com/GoogleCloudPlatform/blueprints/blob/2ea8a3798ad7cdbb7a6411950c69c61e34080a4b/catalog/networking/network/setters.yaml#L12)

Reply:
The prefix is more for the overall network, but I agree we probably need to rework this UX at some point.